packages/samples/headless-ssr/src/components/hydration-metadata.tsx

```diff
@@ -0,0 +1,33 @@
+import {FunctionComponent} from 'react';
+import {SearchCSRState, SearchSSRState} from '../common/engine';
+
+export interface HydrationMetadataProps {
+  initialState: SearchSSRState;
+  hydrationResult?: SearchCSRState;
+}
+
+export const HydrationMetadata: FunctionComponent<HydrationMetadataProps> = ({
+  initialState,
+  hydrationResult,
+}) => (
+  <>
+    <div>
+      Hydrated:{' '}
+      <span id="hydrated-indicator">{hydrationResult ? 'yes' : 'no'}</span>
```

Comment:
nit: guess it could be left as `{hydrationResult}` and then the test checks for `true`?